ts/Core/Renderer/SVG/SVGElement.ts

```diff
@@ -223,7 +223,7 @@ declare global {
             ): void;
             public xGetter(key: string): (number|string|null);
             public yGetter(key: string): (number|string|null);
-            public zIndexSetter(value: number, key: string): boolean;
+            public zIndexSetter(value: number, key?: string): boolean;
```

Comment:
It looks that changing key to optional value is not needed with current solution.